modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -0,0 +1,727 @@
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+=== Show the home database
+
+The home database for the current user can be seen using the command `SHOW HOME DATABASE`.
+
+.Query
+[source, cypher]
+----
+SHOW HOME DATABASE
+----
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name    | type       | aliases | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | constituents |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "neo4j" | "standard" | []      | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | []           |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+=== Filter the listed databases
+
+It is also possible to filter and sort the results by using `YIELD`, `ORDER BY`, and `WHERE`.
+
+.Query
+[source, cypher]
+----
+SHOW DATABASES YIELD name, currentStatus, requestedStatus
+ORDER BY currentStatus
+WHERE name CONTAINS 'o'
+----
+
+In this example:
+
+* The number of columns returned has been reduced with the `YIELD` clause.
+* The order of the returned columns has been changed.
+* The results have been filtered to only show database names containing `'o'`.
+* The results are ordered by the `currentStatus` column using `ORDER BY`.
+
+It is also possible to use `SKIP` and `LIMIT` to paginate the results.
+
+
+.Result
+[role="queryresult"]
+----
++--------------------------------------------+
+| name     | currentStatus | requestedStatus |
++--------------------------------------------+
+| "movies" | "online"      | "online"        |
+| "neo4j"  | "online"      | "online"        |
++--------------------------------------------+
+----
+
+[NOTE]
+====
+Note that for failed databases, the `currentStatus` and `requestedStatus` are different.
+This often implies an error, but **that is not always the case**.
+For example, a database may take a while to transition from `offline` to `online` due to a performing recovery.
+Or, during normal operation, a database's `currentStatus` may be transiently different from its `requestedStatus` due to a necessary automatic process, such as one Neo4j instance copying store files from another.
+The possible statuses are `initial`, `online`, `offline`, `store copying` and `unknown`.
+====
+
+
+[role=enterprise-edition not-on-aura]
+[[manage-databases-create]]
+== Create databases
+
+Databases can be created using `CREATE DATABASE`.
+
```

Comment:
maybe have a sentence about the database name being subject to the naming rules? (especially considering the note below)